spec/lib/connection_spec.rb

```diff
@@ -147,7 +147,7 @@
       expect(auth_header == "APIAuth id123:TQiQIW6vVaDC5jvh99uTNkxIg6Q=" || auth_header == "APIAuth id123:PMWBThkB8vKbvUccHvoqu9G3eVk=").to be_truthy
     end
 
-    if Gem.loaded_specs["api-auth"].version.to_s >= "2.0.0"
+    if  Gem.loaded_specs["api-auth"].present? && Gem.loaded_specs["api-auth"].version.to_s >= "2.0.0"
```

Comment:
OK. It's a bit weird (it passes on my machine and on Travis), but I'm happy to added it if it fixes it for others. I assume you'd ran `bundle install` first before running the specs?